src/HypothesisTests.jl

```diff
@@ -72,32 +72,35 @@ end
 function Base.show{T<:HypothesisTest}(io::IO, test::T)
     println(io, testname(test))
     println(io, repeat("-", length(testname(test))))
+    
+    # utilities for pretty-printing
+    conf_string = string(floor((1 - alpha(test)) * 100, 6)) # limit to 6 decimals in %
+    prettify_detail(label::String, value::Any, len::Int) = # len is max length of label
+        "    " * label * " "^max(len - length(label), 0) * string(value)
 
     # population details
     has_ci = applicable(StatsBase.confint, test)
     (param_name, param_under_h0, param_estimate) = population_param_of_interest(test)
     println(io, "Population details:")
-    println(io, "    parameter of interest:   $param_name")
-    println(io, "    value under h_0:         $param_under_h0")
-    println(io, "    point estimate:          $param_estimate")
+    println(io, prettify_detail("parameter of interest:", param_name, 32))
+    println(io, prettify_detail("value under h_0:", param_under_h0, 32))
+    println(io, prettify_detail("point estimate:", param_estimate, 32))
     if has_ci
-        println(io, "    95% confidence interval: $(StatsBase.confint(test))")
+        println(io, prettify_detail(conf_string*"% confidence interval:", StatsBase.confint(test), 32))
     end
     println(io)
 
     # test summary
-    p = pvalue(test)
-    outcome = if p > 0.05 "fail to reject" else "reject" end
-    tail = default_tail(test)
+    tail = HypothesisTests.tail(test)
+    p = pvalue(test) # obeys value of HypothesisTests.tail(test) if applicable
+    outcome = if p > alpha(test) "fail to reject" else "reject" end
```

Comment:
It's more standard to use `p > alpha(test) ? ... : ...`. Below, since the conditions are relatively long, would be better to repeat `tailvalue = ` inside each branch (i.e. closer to what it was before).